pages/rest_api/pipelines.md.erb

```diff
@@ -247,6 +247,7 @@ Optional [request body properties](/docs/api#request-body-properties):
 <tbody>
   <tr><th><code>env</code></th><td>The pipeline environment variables. <p class="Docs__api-param-eg"><em>Example:</em> <code>{"KEY":"value"}</code></p></td></tr>
   <tr><th><code>provider_settings</code></th><td>The source provider settings. <p class="Docs__api-param-eg"><em>Example:</em> <code>{"provider_settings": { "publish_commit_status": true, "build_pull_request_forks": true }}</code></p></td>
+  <tr><th><code>team_uuids</code></th><td>An array of team UUIDs to add this pipeline to, if your organization has teams enabled. You can find your team’s UUID either via the <a href="/docs/graphql-api">GraphQL API</a>, or on the settings page for a team.<p class="Docs__api-param-eg"><em>Example:</em> <code>{"team_uuids": ["14e9501c-69fe-4cda-ae07-daea9ca3afd3"]}</code></p></td>
```

Comment:
> if your organization has teams enabled

This could go right at the end as its own sentence? Something like 'This property is only available if you organisation has enabled Teams'?
Separating it out takes some of the cognitive load out of the first sentence.